homeassistant/components/daikin_madoka/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""Platform for the Daikin AC."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from pymadoka import Controller, discover_devices, force_device_disconnect
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_DEVICE,
+    CONF_DEVICES,
+    CONF_FORCE_UPDATE,
+    CONF_SCAN_INTERVAL,
+)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
+
+from . import config_flow  # noqa: F401
+from .const import CONTROLLERS, DOMAIN
+
+PARALLEL_UPDATES = 0
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+
+COMPONENT_TYPES = ["climate"]
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
This is a new integration and thus YAML flow can't be used, see [ADR-0010](https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md) for more context.

I see you already marked it a deprecated, what is the reason?

Reply:
I took as an example some old integration from the time both configuration ways were allowed and didn't realize I had to remove YAML support. Removing..